intake_stac/catalog.py

```diff
@@ -152,6 +152,22 @@ def _load(self):
     def _get_metadata(self, **kwargs):
         return kwargs
 
+    def to_geopandas(self):
```

Comment:
I think we should, at a minimum, allow passing the crs parameter in as a kwarg here.

Reply:
I vote we wait to do anything more than this (defaults from catalog metadata). Those sort of features would make for a nice follow up PR.

Reply:
> I think we should, at a minimum, allow passing the crs parameter in as a kwarg here.

What should we have as a default value for `crs`: `None`? `{"init": "epsg:4326"}`?

Reply:
I think I would set the signature as:

```python
def to_geopandas(self, crs=None):
```

but then add the default in the case where crs is None:

```python
if crs is None:
    crs = {"init": "epsg:4326"}
```

Reply:
Done!